server/src/main/java/org/hyperskill/database/serverside/Server.java

```diff
@@ -1,7 +1,49 @@
 package org.hyperskill.database.serverside;
 
+import java.io.*;
+import java.net.*;
+import java.util.*;
+
+//java -cp target/...jar com.kkk.app
+
 public class Server {
+    private static final int PORT = 34522;
+
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        try (ServerSocket server = new ServerSocket(PORT)) {
+            while (true) {
+                Session session = new Session(server.accept());
+                session.run(); // it does not block this thread
```

Comment:
Only it does block it. Well, not really blocks, cause execution still continues, but it only serves one client.
You have another infinite loop in Session::run. So by calling it inside the first infinite loop, its iteration won't end until a client disconnects.

You can fix it by making Session into Thread